vue-components/stories/TextArea.stories.ts

```diff
@@ -28,7 +35,9 @@ export function basic( args: object ): Component {
 
 basic.args = {
     label: 'Label',
-    placeholder: 'Placeholder'
+    placeholder: 'Placeholder',
+    resize: 'vertical',
```

Comment:
I noticed that you sneaked this one in! :-)

Reply:
mwahaha :japanese_goblin: yeah it was causing some annoying console warnings on each re-render